packages/react-layouts/src/Breadcrumb/index.js

```diff
@@ -12,18 +12,27 @@ import styled from '@emotion/styled/macro';
 import { textStyles } from '@quid/theme';
 import { sizes } from '@quid/theme';
 
+type BasicProps = {
+  path: string,
+  label: React$Node,
+  disabled?: boolean,
+  external?: boolean,
+  emphasized?: boolean,
+};
+
+export type ContentProps = BasicProps & {
```

Comment:
May you try `{ ...BasicProps, index: number }` instead? It's usually more reliable